radsat-sk/operation/subsystems/RAntenna.c

```diff
@@ -70,7 +71,7 @@ int antennaDeploymentAttempt(void) {
 	while ( antennaDeploymentAttempts < MAX_DEPLOYMENT_ATTEMPTS ) {
 
 		// Get status of the A side antenna
-		error = IsisAntS_getStatusData(ANTENNA_I2C_SLAVE_ADDR_PRIMARY,isisants_sideA,&RISISantsStatus);
+		error = IsisAntS_getStatusData(ANTENNA_INDEX,isisants_sideA,&RISISantsStatus);
```

Comment:
Space after comma. Same style issue on line 95, 105, 121, 131, 153, 174, 184, 200, 210, 247, and 266. May as well fix it while you are there but I won't black this PR based on this.